### DISCUSSION TITLE
WebRTC stack upgrade

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5797] [General]
Hello,
I see there is an update for the WebRTC stack for Ant Media Server version 2.7.0 as per the release logs [https://github.com/ant-media/Ant-Media-Server/releases](url)

Which version of the WebRTC stack is currently AMS running on?

Thank you.

**Comment by mekya** [comment]
Hi @Mohit-3196,

Thank you for asking the question. Ant Media Server v2.7.0 uses the WebRTC stack m108. We use the original webrtc stack in the google repository. 
https://webrtc.googlesource.com/src/

Cheers
Oguz
